chrome/content/tabutils.js

```diff
@@ -697,7 +697,7 @@ tabutils._tabOpeningOptions = function() {
     if ((function() {
       switch (TU_getPref("extensions.tabutils.openTabNext", 1)) {
         case 1: //All
-        case 2: return aRelatedToCurrent || aReferrerURI || (aURI != "about:blank" && aURI != "about:newtab"); //All but New Tab
+        case 2: return aRelatedToCurrent || aReferrerURI || !isBlankPageURL(aURI); //All but New Tab
```

Comment:
Again (lost from Github's review process): Good catch! It utilizes the `gInitialPages` in `isBlankPageURL` function.